styles.css

```diff
@@ -589,3 +589,105 @@ ul.list-items.skill-items {
 .button-5 {
   padding-left: 10px;
 }
+
+form {
+  display: block;
+  margin: 0;
+}
+
+.form-class {
+  display: flex;
+  flex-direction: column;
+  margin-top: 12px;
+  align-items: center;
+}
+
+.form-btn {
+  color: #6070ff;
+  background-color: #fff;
```

Comment:
- [ ] Please increase your font-weight and add button interactions as in the figa design:
![image](https://user-images.githubusercontent.com/91486791/220391740-af0c116b-e112-4cd0-ba3e-6ef0ae527d59.png)
![image](https://user-images.githubusercontent.com/91486791/220391801-06ecf87e-370e-45d5-8540-aecdcf504f76.png)
your button: 
![image](https://user-images.githubusercontent.com/91486791/220391949-1148d29e-eaa8-4796-b0a7-ef6d412764e0.png)